tests/unit/scripting.tcl

```diff
@@ -616,6 +616,34 @@ start_server {tags {"scripting"}} {
              [run_script {return redis.sha1hex('Pizza & Mandolino')} 0]
     } {da39a3ee5e6b4b0d3255bfef95601890afd80709 74822d82031af7493c20eefa13bd07ec4fada82f}
 
+    test "redis.monotonic() measures elapsed time" {
+        set escaped [run_script {
+            local start = redis.monotonic()
+            while redis.monotonic() - start < 1000000 do end
+            return {double = redis.monotonic() - start}
+        } 0]
+        assert_morethan_equal $escaped 1000000 ;# 1 second (1 second = 1000000 microseconds)
+    }
+
+    test "os.clock() measures elapsed time" {
+        set escaped [run_script {
+            local start = os.clock()
+            while os.clock() - start < 1 do end
+            return {double = os.clock() - start}
+        } 0]
+        assert_morethan_equal $escaped 1 ;# 1 second
+    }
+
+    test "Prohibited dangerous lua methods in sandbox" {
+        assert_error {ERR *attempt to call field 'execute'*} {run_script {os.execute()} 0}
+        assert_error {ERR *attempt to call field 'exit'*} {run_script {os.exit()} 0}
+        assert_error {ERR *attempt to call field 'getenv'*} {run_script {os.getenv()} 0}
+        assert_error {ERR *attempt to call field 'remove'*} {run_script {os.remove()} 0}
+        assert_error {ERR *attempt to call field 'rename'*} {run_script {os.rename()} 0}
+        assert_error {ERR *attempt to call field 'setlocale'*} {run_script {os.setlocale()} 0}
+        assert_error {ERR *attempt to call field 'tmpname'*} {run_script {os.tmpname()} 0}
+    }
```

Comment:
Maybe we can write a test that verify that `os` only contains the `clock` function?

Reply:
Done with https://github.com/redis/redis/pull/12971/commits/7e2d5c9eca0035f88f815e20fcdaa2b0a6745b47